src/components/Header/components/PeopleSearch/people-search.scss

```diff
@@ -40,8 +40,29 @@
   }
 
   .no-results {
```

Comment:
Maybe add a comment explaining that this shows loading for 350ms, then no results, and a word or two about how (opacity).